lib/siwapp/customers.ex

```diff
@@ -75,9 +75,17 @@ defmodule Siwapp.Customers do
   Gets a customer by id
   """
   @spec get!(binary | integer) :: Customer.t()
-  def get!(id), do: Repo.get!(Customer, id)
+  def get!(id) do
+    with nil <- Repo.get(Customer, id),
+         do: raise(Siwapp.Error.NotFoundError, id: id, type: "customer")
```

Comment:
If this method is used in Api enpoint what would happen ? Dunno if siwapp use grapql or API rest